tests/series/test_utf8_ops.py

```diff
@@ -582,3 +582,49 @@ def test_series_utf8_extract_all_bad_pattern() -> None:
     pattern = Series.from_arrow(pa.array(["["]))
     with pytest.raises(ValueError):
         s.str.extract_all(pattern)
+
+
+@pytest.mark.parametrize(
+    ["data", "substrs", "expected"],
+    [
+        # No broadcast
+        (["foo", "barbaz", "quux"], ["foo", "baz", "baz"], [0, 3, -1]),
+        # Broadcast substrs
+        (["foo", None, "quux"], ["foo"], [0, None, -1]),
+        # Broadcast data
+        (["foo"], ["foo", None, "baz"], [0, None, -1]),
+        # Broadcast null data
+        ([None], ["foo", "bar", "baz"], [None, None, None]),
```

Comment:
I feel like this case actually breaks, and we likely have a returned Series of type `String` instead of `Int64` because the `FullNull` constructor is probably a little too loose. Can we verify that?

Reply:
fixed it, weird that tests were passing